components/inputs/docs/styling-native-inputs.md

```diff
@@ -0,0 +1,153 @@
+# Styling Native Inputs
+
+In the majority of cases, it's recommended to use our web component wrappers around native inputs like text, textarea, checkboxes and radios. They'll provide functionality like inline help, skeletons, form participation and localized validation out-of-the-box.
+
+In the rare circumstances where it's preferred to use a native HTML input instead, it is possible to apply styles such that they appear similar to our web component variants.
+
+## Checkbox Input
+
+Import `input-checkbox-styles.js` and apply the `d2l-input-checkbox` CSS class to the input:
+
+<!-- docs: demo code display:block -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { checkboxStyles } from '@brightspace-ui/core/components/inputs/input-checkbox.js';
+
+  class MyCheckboxElem extends LitElement {
+
+    static get styles() {
+      return checkboxStyles;
+    }
+
+    render() {
+      return html`<input type="checkbox" class="d2l-input-checkbox">`;
+    }
+
+  }
+  customElements.define('d2l-my-checkbox-elem', MyCheckboxElem);
+</script>
+<d2l-my-checkbox-elem></d2l-my-checkbox-elem>
+```
+
+## Radio Inputs
+
+### With Labels
+
+The simplest way to apply radio styles is to use the `d2l-input-radio-label` CSS class on a `<label>` element that wraps the input.
+
+For disabled items, add the `d2l-input-radio-label-disabled` class on the label and the `disabled` attribute on the input itself.
+
+<!-- docs: demo code display:block -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { radioStyles } from '@brightspace-ui/core/components/inputs/input-radio-styles.js';
+
+  class MyRadioElem extends LitElement {
+
+    static get styles() {
+      return radioStyles;
+    }
```

Comment:
Is it more likely/useful to pretend they also have their own styles here?

Reply:
Eh, I guess we don't really do `...` pseudo code stuff in these and we don't need to overcomplicate it with real styles.